CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# Foodsoft 4.7.1
+(31 December 2020)
+
+* Fix minor navigation glitch ([78c4ebc](https://github.com/foodcoops/foodsoft/commit/78c4ebcb4b61891e13d3dbb9f07c34ad88c41f4a))
+* Fix legacy tasks without user ([beaca7d](https://github.com/foodcoops/foodsoft/commit/beaca7d384bf9c9a9c5275ac7d81bb012374372e))
+* Fix GroupOrder update timestamp ([3d5043f](https://github.com/foodcoops/foodsoft/commit/3d5043fd052f79b33f8cae2ea3a5cc81ee972ba5))
+* Move from Travis CI to GitHub Actions ([9b092ee](https://github.com/foodcoops/foodsoft/commit/9b092ee362df8a46854d8dc42125963f86dadf2f))
```

Comment:
The changelog is for users, developers can look at the git commits.

Reply:
Are you suggesting to remove this item (GitHub Actions), or are you suggesting to remove the links?

Reply:
To remove this line. The changelog is meant more for users.